subgraphs/_reference_/src/sdk/protocols/bridge/pool.ts

```diff
@@ -0,0 +1,638 @@
+  /**
+   * Adds a given amount to the pool's mintSupply. It should only be used for pools of type BURN_MINT.
+   * @param amount {BigInt} The amount to add to the pool's mintSupply. It can be positive or negative.
+   */
+  addMintSupply(amount: BigInt): void {
+    this.pool.mintSupply = this.pool.mintSupply!.plus(amount);
+    this.save();
+  }
+
+  /**
+   * Adds a given amount to the pool's outputTokenSupply. It should only be used for pools
+   * of type LIQUIDITY. Or pools that emit some kind of LP token on deposit.
+   * @param amount
+   */
+  addOutputTokenSupply(amount: BigInt): void {
+    if (!this.pool.outputTokenSupply) {
+      this.pool.outputTokenSupply = BIGINT_ZERO;
+    }
+    this.pool.outputTokenSupply = this.pool.outputTokenSupply!.plus(amount);
+    this.save();
+  }
+
+  /**
+   * Sets the pool's outputTokenSupply value. It should only be used for pools
+   * of type LIQUIDITY. Or pools that emit some kind of LP token on deposit.
+   * It will also update the outputTokenPriceUSD value.
+   * @param amount
+   */
+  setOutputTokenSupply(amount: BigInt): void {
+    this.pool.outputTokenSupply = amount;
+    this.refreshOutputTokenPriceUSD();
+    this.save();
+  }
+
+  /**
+   * Adds a given amount to the pool's stakedOutputTokenAmount.
+   * @param amount
+   * @returns
+   */
+  addStakedOutputTokenAmount(amount: BigInt): void {
+    if (!this.pool.stakedOutputTokenAmount) {
+      this.pool.stakedOutputTokenAmount = BIGINT_ZERO;
+      this.save();
+      return;
+    }
+
+    this.pool.stakedOutputTokenAmount =
+      this.pool.stakedOutputTokenAmount!.plus(amount);
+    this.save();
+  }
+
+  /**
+   * Sets the pool's stakedOutputTokenAmount value.
+   * @param amount
+   */
+  setStakedOutputTokenAmount(amount: BigInt): void {
+    this.pool.stakedOutputTokenAmount = amount;
+    this.save();
+  }
+
+  /**
+   * Updates the price of the pool's output token in USD.
+   * This is automatically called when changing the output token supply via setOutputTokenSupply
+   * but can be called manually if necessary.
+   */
+  refreshOutputTokenPriceUSD(): void {
+    if (!this.pool.outputToken) {
+      return;
+    }
+    const token = this.tokens.initToken(this.pool.outputToken);
+    const price = this.protocol.pricer.getTokenPrice(token);
+
+    this.pool.outputTokenPriceUSD = price;
+    this.save();
+  }
+
+  /**
+   * Sets the rewardTokenEmissions and its USD value for a given reward token.
+   * It will also create the RewardToken entity and add it to the pool rewardTokens array
+   * if not already present.
+   * @param type The type of reward token
+   * @param token The actual token being rewarded
+   * @param amount The daily amount of reward tokens emitted to this pool
+   */
+  setRewardEmissions(
+    type: RewardTokenType,
+    token: Token,
+    amount: BigInt
+  ): void {
+    const rToken = this.tokens.initRewardToken(type, token);
+    const amountUSD = this.protocol.pricer.getAmountPrice(token, amount);
```

Comment:
`getAmountPrice` -> `convertAmountToUSD`?

Reply:
I renamed it instead to `getAmountValueUSD`